src/views/mfa-verify/inputs/PasswordBox.js

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+    'okta',
+    'views/shared/TextBox'
+  ],
+  function (Okta, TextBox) {
+
+    return TextBox.extend({
+      template: Okta.tpl('' +
+        '<input type="password" placeholder="{{placeholder}}" name="{{name}}" id="{{inputId}}" value="{{value}}"/>' +
+        '<span class="password-toggle">' +
+        '<span class="button button-on" style="background:#929292; border-color:#929292; color:#ffffff;">' +
+        '{{i18n code="mfa.challenge.answer.showAnswer" bundle="login"}}</span>' +
+        '<span class="button button-off">{{i18n code="mfa.challenge.answer.hideAnswer" bundle="login"}}</span>' +
+        '</span>'),
+
+      constructor: function () {
+        TextBox.apply(this, arguments);
+
+        this.events['click .password-toggle .button-on'] = '_showPassword';
+        this.events['click .password-toggle .button-off'] = '_hidePassword';
+      },
+
+      changeType: function (type) {
+        this.$(':input').first().prop('type', type);
```

Comment:
Here we should at least be calling the inputs changeType method (as it was originally) so we can get the type + options behavior.